core/src/test/java/org/apache/iceberg/TestScanPlanningAndReporting.java

```diff
@@ -61,16 +62,16 @@ public void scanningWithMultipleDataManifests() throws IOException {
 
     assertThat(scanReport.tableName()).isEqualTo(tableName);
     assertThat(scanReport.snapshotId()).isEqualTo(2L);
-    assertThat(scanReport.scanMetrics().totalPlanningDuration().totalDuration())
-        .isGreaterThan(Duration.ZERO);
-    assertThat(scanReport.scanMetrics().resultDataFiles().value()).isEqualTo(3);
-    assertThat(scanReport.scanMetrics().resultDeleteFiles().value()).isEqualTo(0);
-    assertThat(scanReport.scanMetrics().scannedDataManifests().value()).isEqualTo(2);
-    assertThat(scanReport.scanMetrics().skippedDataManifests().value()).isEqualTo(0);
-    assertThat(scanReport.scanMetrics().totalDataManifests().value()).isEqualTo(2);
-    assertThat(scanReport.scanMetrics().totalDeleteManifests().value()).isEqualTo(0);
-    assertThat(scanReport.scanMetrics().totalFileSizeInBytes().value()).isEqualTo(30L);
-    assertThat(scanReport.scanMetrics().totalDeleteFileSizeInBytes().value()).isEqualTo(0L);
+    ScanMetricsResult result = scanReport.scanMetrics();
+    assertThat(result.totalPlanningDuration().totalDuration()).isGreaterThan(Duration.ZERO);
+    assertThat(result.resultDataFiles().value()).isEqualTo(3);
+    assertThat(result.resultDeleteFiles().value()).isEqualTo(0);
+    assertThat(result.scannedDataManifests().value()).isEqualTo(2);
+    assertThat(result.skippedDataManifests().value()).isEqualTo(0);
+    assertThat(result.totalDataManifests().value()).isEqualTo(2);
+    assertThat(result.totalDeleteManifests().value()).isEqualTo(0);
+    assertThat(result.totalFileSizeInBytes().value()).isEqualTo(30L);
+    assertThat(result.totalDeleteFileSizeInBytes().value()).isEqualTo(0L);
```

Comment:
@nastra, why not add an assertion for the skipped manifest counters in existing tests?

Reply:
I realized that those were missing in the existing tests when working on https://github.com/apache/iceberg/pull/5792 and included/added them there